### ISSUE TITLE
Dashboard links are wrong in JupyterLab but not Jupyter

### ISSUE THREAD

**Issue by jlewi** [kubeflow/pipelines#1880]
**What happened:**

I ran this [notebook](https://github.com/jlewi/examples/blob/898dca20438c85b77422f1d05afd8daa51960d30/xgboost_synthetic/build-train-deploy.ipynb) inside a Kubeflow Jupyter Lab instance running GKE with IAP.

The notebook submits a pipeline using the KFP client which prints out the links to view the pipeline in the pipelines UI.

When I run this in JupyterLab the links are invalid.

Here's the link it gives for the run

```
https://jlewi-0-6-2-rc.endpoints.jlewi-dev.cloud.goog/notebook/kubeflow-jlewi/jlewi-nb/files/pipeline#/runs/details/cec4eed4-c136-11e9-ab37-42010a8e00c9
```
  * This link is not correct; its adding the pipelines UI path to the jupyter lab path

Running the same code in jupyter produces valid links; here's the link when running in Jupyter

```
https://jlewi-0-6-2-rc.endpoints.jlewi-dev.cloud.goog/pipeline/#/runs/details/bf3571e5-c136-11e9-ab37-42010a8e00c9
```
  * pipelines is a subpath of the host

kfp.__version__ isn't defined so I'm not sure how to explicitly check which version I'm using.

But I should be installing this version
https://storage.googleapis.com/ml-pipeline/release/0.1.20/kfp.tar.gz

**Comment by Ark-kun** [comment]
This issue is duplicate of https://github.com/kubeflow/pipelines/issues/252
The root cause is a bug in JupyterLab that cannot be worked around on KFP side https://github.com/jupyterlab/jupyterlab/issues/5211. The JupyterLab team has finally fixed the bug several months ago. https://github.com/jupyterlab/jupyterlab/pull/5814
Couple of months ago I've created an issue to update the JupyterLab version in Kubeflow: https://github.com/kubeflow/kubeflow/issues/3442

**Comment by Ark-kun** [comment]
@paveldournov No triaging issues at night... on vacation =)

**Comment by Ark-kun** [comment]
I've verified that the issue is fixed in KF 0.7
